Purchases/Networking/RCBackend.h

```diff
@@ -11,25 +11,16 @@
 
 #import "RCPurchases.h"
 #import "RCSubscriberAttribute.h"
+#import "RCProductInfo.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @class RCPurchaserInfo, RCHTTPClient, RCIntroEligibility, RCPromotionalOffer, RCSubscriberAttribute, RCSystemInfo;
 
-typedef NS_ENUM(NSInteger, RCPaymentMode) {
-    RCPaymentModeNone = -1,
-    RCPaymentModePayAsYouGo = 0,
-    RCPaymentModePayUpFront = 1,
-    RCPaymentModeFreeTrial = 2
-};
-
 extern NSErrorUserInfoKey const RCSuccessfullySyncedKey;
 extern NSString * const RCAttributeErrorsKey;
 extern NSString * const RCAttributeErrorsResponseKey;
 
-API_AVAILABLE(ios(11.2), macos(10.13.2))
-RCPaymentMode RCPaymentModeFromSKProductDiscountPaymentMode(SKProductDiscountPaymentMode paymentMode);
```

Comment:
moved to `RCProductInfo`.

I'm not a huge fan of the format, I was going to turn it into a static method instead but changing it didn't seem to add much value anyway